extensions/geode-modules/src/main/java/org/apache/geode/modules/util/BootstrappingFunction.java

```diff
@@ -87,6 +91,11 @@ private Cache verifyCacheExists() {
     return cache;
```

Comment:
I think this would be a good opportunity to eliminate the `CacheFactory.getAnyInstance()` call above. The cache should be available through the FunctionConext.

Reply:
I'm going to open a separate Jira for that - it seems a bit orthogonal to what's in this PR.